spec/requests/api/v3/work_package_resource_spec.rb

```diff
@@ -259,7 +259,7 @@
       shared_examples_for 'lock version updated' do
         it {
           expect(subject.body).to be_json_eql(work_package.reload.lock_version)
-            .at_path('lockVersion')
+                                    .at_path('lockVersion')
```

Comment:
Again weird indentation. I'd put the `.to be_json_eql` in an extra line before or move the `.at_path` into the same line.